x-pack/plugins/cases/public/components/user_actions/property_actions.tsx

```diff
@@ -60,10 +76,19 @@ const UserActionPropertyActionsComponent = ({
   }, []);
 
   const propertyActions = useMemo(() => {
-    const showEditPencilIcon = permissions.update;
-    const showTrashIcon = permissions.delete && deleteLabel && onDelete;
-    const showQuoteIcon = permissions.create;
-    const showLensEditor = permissions.update && canUseEditor && actionConfig;
+    const showEditPencilIcon = permissions.update && actions.includes(totalActions.edit);
```

Comment:
Do we need to wrap this in a `Boolean`? Or is that one for expressions that have a callback check for defined?

Reply:
Both the `permissions.update` and the `actions.includes` returns a `boolean` so `showEditPencilIcon` will always be a `boolean`. On the `showTrashIcon` variable the logic clause is a mix of booleans, strings, and functions. If they are defined the `showTrashIcon` can be a string or a function instead of a `boolean`. This may introduce bugs. Imagine if the label is empty but the user has permissions to delete. Then it will hide the trash icon. We do not want that.